src/common/integration/urlProcessing.js

```diff
@@ -6,7 +6,7 @@ import { BookingState } from '../../context/BookingState';
 
 type Props = {
   wasSelected: boolean,
-  setSelected: () => void,
+  setSelected: (bookingId: string) => void,
```

Comment:
Shouldn't we change the function signature of `App`'s `urlBookingSelected` method? Actually the typecheck should fail here ideally, shouldn't it? :thinking: